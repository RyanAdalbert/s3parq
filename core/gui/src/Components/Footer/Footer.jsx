import React from 'react';
import styled from 'styled-components';

const FooterSec = styled.section`
  display: flex;
  justify-content: flex-end;
  background: #ccc;
  padding: 2px 5%;

  p {
    margin: 10px;
  }
`;

const Footer = () => (
  <FooterSec>
    <p>
      &copy; {new Date().getFullYear()}{' '}
      <a href="https://www.integrichain.com/">IntegriChain</a> Incorporated. All
      Rights Reserved.
    </p>
  </FooterSec>
);

export default Footer;
