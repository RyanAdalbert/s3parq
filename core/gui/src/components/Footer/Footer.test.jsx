import Footer from './Footer';

describe('<Footer /> rendering', () => {
  const wrapper = shallow(<Footer />);
  it('should render one <FooterSec/>', () => {
    expect(wrapper.find('p').length).toEqual(1);
  });
});
