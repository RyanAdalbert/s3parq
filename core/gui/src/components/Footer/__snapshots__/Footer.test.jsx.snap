// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> rendering should render one <FooterSec/> 1`] = `
<styled.section>
  <p>
    Â© 
    2019
     
    <a
      href="https://www.integrichain.com/"
    >
      IntegriChain
    </a>
     Inc. All Rights Reserved.
  </p>
</styled.section>
`;
